tmt/steps/provision/artemis.py

```diff
@@ -102,6 +111,12 @@ def _normalize_log_type(key_address: str, raw_value: Any, logger: tmt.log.Logger
     raise tmt.utils.NormalizationError(key_address, raw_value, 'a string or a list of strings')
 
 
+def _format_log_blobs(blobs: list['GuestLogBlobType']) -> str:
+    sorted_blobs = sorted(blobs, key=lambda blob: blob['ctime'])
+    parts = [f"{blob['ctime']}\n{blob['content']}" for blob in sorted_blobs]
+    return "\n\n".join(parts)
```

Comment:
I dislike the chain of single-use variables. How about the following expression?

```python
return "\n\n".join(
    f"{blob['ctime']}\n{blob['content']}" for blob in sorted(blobs, key=lambda blob: blob['ctime'])
)
```